fossilize_external_replayer_linux.hpp

```diff
@@ -418,7 +418,7 @@ bool ExternalReplayer::Impl::kill()
 		kill_fd = -1;
 	}
 
-	bool ret = killpg(pid, SIGTERM) == 0;
+	bool ret = killpg(pid, SIGKILL) == 0;
```

Comment:
I'm not sure if this is enough because the leaking processes in stopped state didn't go away when I sent SIGKILL manually, it needed an additional SIGCONT. But let's see how it changes once this update is deployed.

Reply:
I tested this and SIGKILLing a stopped child worked just fine.